pkg/web/apps/apps_service.go

```diff
@@ -29,7 +28,7 @@ func (a *appsService) GetApps(c echo.Context) (*[]models.App, error) {
 	apps, err := a.psqlRepository.GetApps(c)
 
 	if err != nil {
-		return nil, httperrors.NotFound(c, "No apps found")
+		return nil, models.ErrNotFound
```

Comment:
Is not this the case that we in common sense agreed that should return the 204? Is not here the reference to the endpoint /apps/?